### DISCUSSION TITLE
Unable to Delete Configmaps

### DISCUSSION THREAD

**Discussion by VUMonkey** [mia-platform/community#399] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

12.2.0

### Description

Under `Microservices` -> `ConfigMaps and Secrets` I am unable to delete ConfigMaps I've created. I removed any associated files, but still no luck. There's a javascript error that mentions not being able to read the properties of undefined (reading "name"). Has anybody else run into this? Is there a solution or workaround?

Thanks,

![image](https://github.com/mia-platform/community/assets/143722005/1463d3e9-1d60-4316-a6bf-dce16f54299d)

### Actual Outcome

Nothing happens except a new error appears in the javascript console:
![image](https://github.com/mia-platform/community/assets/143722005/1463d3e9-1d60-4316-a6bf-dce16f54299d)

### Expected Outcome

A configmap is deleted/removed when I click the `Delete ConfigMap` button

**Comment by danielemarostica** [comment, marked as answer]
Hi @VUMonkey,

Thank you for reaching out.
Unfortunately there are no workarounds for the issue, but good news is we have already worked out a fix and will bring it in prod with the next release.

I'll keep you updated on this issue

**Comment by danielemarostica** [comment]
Hi @VUMonkey,

I'm here to inform you that we just released the fix in production. You should be now able to delete configmaps again.
Here the [release note](https://docs.mia-platform.eu/docs/release-notes/v12.2.1#bug-fix)

In case of any doubt please let me know.
